frontend/public/features.ts

```diff
@@ -128,11 +129,36 @@ const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
     }
   );
 
+const monitoringConfigMapPath = `${k8sBasePath}/api/v1/namespaces/monitoring/configmaps/sharing-config`;
+const detectMonitoringURLs = dispatch => coFetchJSON(monitoringConfigMapPath)
+  .then(
+    res => {
+      const {alertmanagerHost, grafanaHost, prometheusHost} = res.data;
+      // The ConfigMap does not provide URL protocols, so we assume they are the same as Console
+      const protocol = location.protocol;
```

Comment:
I'm wondering if it's better to hard-code to https. Do we need http for development?

Reply:
The ConfigMap is now providing the full URL.